### PR TITLE
front: Add token propagation from App.js

### PR DESCRIPTION
**Pull request by No311**
As authentication is not done, it is currently done with a dummy token "dummy".

### CODE REVIEW

File: front/src/BottomBar/index.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const getConnectionStatusBackground = type => {
  */
 
 const BottomBar = props => {
-  const { changeActivePage, activePage, connectionStatus } = props;
+  const { changeActivePage, activePage, connectionStatus, token } = props;
```
Comment (g868) [inline front/src/BottomBar/index.js]: Please add "token" to the props in the documentation

Comment (g868) [inline front/src/BottomBar/index.js]: "connectionStatus" should replace "isConnected" in the props documentation

File: front/src/RefboxField/index.js [hunk 2]
```diff
@@ -44,7 +44,11 @@ const RefboxField = props => {
               id={`${tag}_magenta`}
               onClick={() => {
                 console.log(`${TAG_VALUES[tag]}_magenta`);
-                sendToServer(`${TAG_VALUES[tag]}_magenta`, "command");
+                sendToServer(
+                  `${TAG_VALUES[tag]}_magenta`,
+                  "command",
+                  props.token
```
Comment (g868) [inline front/src/RefboxField/index.js]: Please add "token" to the props in the documentation

File: front/src/Settings/index.js [hunk 3]
```diff
@@ -31,15 +31,15 @@ const COMMAND_VALUES = {
 };
 
 const Settings = props => {
-  const { turtles } = props;
+  const { turtles, token } = props;
```
Comment (g868) [inline front/src/Settings/index.js]: Please add "token" to the props in the documentation

File: front/src/BottomBar/index.js (hunk 1 above)
Comment (No311) [reply front/src/BottomBar/index.js]: done

Comment (No311) [reply front/src/BottomBar/index.js]: done

File: front/src/RefboxField/index.js (hunk 2 above)
Comment (No311) [reply front/src/RefboxField/index.js]: done

File: front/src/Settings/index.js (hunk 3 above)
Comment (No311) [reply front/src/Settings/index.js]: done

**Comment by No311** [comment]
I'll look at it when I couple authentication to it, but I don't quite think so.

**Pull request merged**